internal/service/searchindex/resource_search_index_test.go

```diff
@@ -437,8 +442,9 @@ const (
 	with           = true
 	without        = false
 
-	analyzersTF      = "\nanalyzers = <<-EOF\n" + analyzersJSON + "\nEOF\n"
-	mappingsFieldsTF = "\nmappings_fields = <<-EOF\n" + mappingsFieldsJSON + "\nEOF\n"
+	analyzersTF                = "\nanalyzers = <<-EOF\n" + analyzersJSON + "\nEOF\n"
+	incorrectFormatAnalyzersTF = "\nanalyzers = <<-EOF\n" + incorrectFormatAnalyzersJSON + "\nEOF\n"
```

Comment:
do you think it would be also to have an acc test or unit tests should be enough in this case?

Reply:
I had that question for myself. the cost is minimum since it doesn't call the API but fails before. The reason why I opted for acc test is because in the end I could re-use an existing one by just adding a further step, but unit test would have worked as well. 

Let me know if you have a strong opinion on this.

Reply:
it's fine with unit test only